app.js

```diff
@@ -74,6 +75,7 @@ app.get("/", (req, res) => {
 
 app.use("/api/v1/users", userRouter); //users route
 app.use("/api/v1/universities", universityRouter); //univesity route
+app.use("/api/v1/department", departmentRouter); //univesity route
```

Comment:
let use be consistent with the pluralization. Please add ass to department`app.use("/api/v1/departments", departmentRouter);`